models/issue.go

```diff
@@ -166,6 +166,18 @@ func (issue *Issue) LoadAttributes() error {
 	return issue.loadAttributes(x)
 }
 
+// GetIsRead load the `IsRead` field of the issue
+func (issue *Issue) GetIsRead(userID int64) error {
```

Comment:
The `IsRead` field is still used (in `/templates/repo/issue/list.tmpl`).

While it might make sense to eventually remove the is_read column, it seems outside the scope of this PR.